packages/components/src/internal/components/samples/SampleAssayDetail.tsx

```diff
@@ -197,25 +197,15 @@ export const SampleAssayDetailBodyImpl: FC<SampleAssayDetailBodyProps & Injected
 
     const { queryModelsWithData, tabOrder } = useMemo(() => {
         const models: Record<string, QueryModel> = {};
```

Comment:
getSampleAssayDetailEmptyText currently assumes aliquot filters only. Needs to be updated to know if the blank result is due to aliquot filters or other filters or both.